sotodlib/io/imprinter.py

```diff
@@ -277,6 +276,96 @@ def register_books(self, bids, obs_lists, commit=True, session=None):
             self.register_book(session, bid, obs_list, commit=False)
         if commit: session.commit()
 
+    def register_hk_books(self, commit=True, session=None):
+        """Register housekeeping books to database
+
+        """
+        session = session or self.get_session()
+
+        # fixme: tel_tube and daq_node may not be the same
+        for daq_node in self.config['sources']: 
```

Comment:
It is each daq-node that has an aggregator running on it. Should be one for each telescope and one for the whole site. (following the folder layout here: https://simonsobs.atlassian.net/wiki/spaces/PRO/pages/88244229/Site+Data+Storage+Outline#Level-2-Smurf-Data-%2F-HK-Data)